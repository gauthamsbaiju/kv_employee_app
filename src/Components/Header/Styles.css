.header{
    background-color: white;
    position: fixed ;
    top: 0px;
    right: 0px;
    width: 100%;
    height: 77px;
    /* background-color: red; */
}

.logo-wrapper{

    display: flex;
    align-items: center;
    justify-content: center;
    
    top: 0px;
    left: 0px;
    width: 300px;
    height: 77px;
    
    /* UI Properties */
    
    background: #E2E8FA40 0% 0% no-repeat padding-box;
    /* opacity: 0.2; */
}

.logo{
    top: 16px;
    left: 46px;
    width: 208px;
    height: 45px;
    
    /* UI Properties */
    
    /* background: transparent url('/public/assets/img/kv-logo.png') 0% 0% no-repeat padding-box;
    opacity: 1; */
}