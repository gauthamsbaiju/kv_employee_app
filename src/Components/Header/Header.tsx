import React from 'react';
import './Styles.css';

// type InputPropsType = {
//   label: string;
//   className: string;
//   onClick: (e) => void;
//   type: 'submit';
// };

const Header: React.FC = () => {
  return (
    <div className='header' data-testid='header-test'>
      <div className='logo-wrapper'>
        <img src='/assets/img/kv-logo.png' className='logo'></img>
      </div>
    </div>
  );
};

export default Header;
