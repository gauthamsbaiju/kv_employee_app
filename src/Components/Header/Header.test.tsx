import { render, screen } from '@testing-library/react';
import Header from './Header';
import '@testing-library/jest-dom';

describe('Test for header component', () => {
  test('Test for rendered snapshot', () => {
    render(<Header />);
    const element = screen.getByTestId('header-test');

    expect(element).toMatchSnapshot();
  });

  test('Test for rendered elements', () => {
    render(<Header />);
    const element = screen.getByTestId('header-test');

    expect(element).toBeInTheDocument();
  });
});
