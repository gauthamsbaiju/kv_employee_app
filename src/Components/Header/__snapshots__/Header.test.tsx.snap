// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for header component Test for rendered snapshot 1`] = `
<div
  class="header"
  data-testid="header-test"
>
  <div
    class="logo-wrapper"
  >
    <img
      class="logo"
      src="/assets/img/kv-logo.png"
    />
  </div>
</div>
`;
